package.json

```diff
@@ -64,13 +64,12 @@
     "crypto": "^1.0.1",
     "dotenv": "^16.0.0",
     "express": "4.17.3",
-    "log4js": "6.4.5",
     "matrix-js-sdk": "^26.1.0",
     "request": "^2.88.2",
     "sqlite3": "^5.1.2"
   },
   "devDependencies": {
-    "@eng-automation/js": "^0.0.22",
+    "@eng-automation/js": "^0.0.23",
```

Comment:
While we're at it - looks like this should be a non-dev dependency.

Reply:
Definitely!